test/extensions/filters/http/cache/http_cache_test.cc

```diff
@@ -296,7 +296,7 @@ TEST_F(LookupRequestTest, PragmaNoFallback) {
 TEST(HttpCacheTest, StableHashKey) {
   Key key;
   key.set_host("example.com");
-  ASSERT_EQ(stableHashKey(key), 9582653837550152292u);
+  ASSERT_EQ(stableHashKey(key), 6153940628716543519u);
```

Comment:
I suppose this PR represents a cache flush to anyone that's got a file-cache stored. Is t hat right? If so it's probably mostly you :)

Reply:
Yes.

Reply:
Why ASSERT_EQ rather than EXPECT_EQ? Usually I use ASSERT_EQ only if, when it fails, that might result in a crash later in the function, like 
```
ASSERT_EQ(5, array.size());
EXPECT_EQ("foo", array[4]);
```

no big deal though